app/controllers/application_controller.rb

```diff
@@ -17,7 +17,7 @@ class ApplicationController < ActionController::Base
 
   helper_method :current_user
   helper_method :oauth_token
-  helper_method :preferred_langauges
+  helper_method :preferred_languages
```

Comment:
?

Reply:
The spell-checker itself definitely wouldn't.
The incredibly naive tool I (wrote and) am using to apply corrections wouldn't.
But there is a dumb human involved (me), and I should know to look to see if the correction results in a duplicate.

Thanks, I'll drop this now.